tests/image/test_deploy.py

```diff
@@ -81,6 +81,7 @@ def test_cli_insecure_registry(list_mocked, post_mocked, get_mocked):
 
 @pytest.fixture
 def mocked_post(monkeypatch):
+    """Helper fixture to mock requests.post used for initial deploy request."""
```

Comment:
I still wouldn't know what the fixture does after reading this ;) Maybe `Mock requests.post to return a status 200 response with location header`?